examples/official-site/sqlpage/migrations/01_documentation.sql

```diff
@@ -757,6 +757,69 @@ and `shell.json` would be placed at the website''s root and contain the followin
 ```
 ', NULL);
 
+INSERT INTO component(name, icon, description, introduced_in_version) VALUES
+    ('loader-start', 'refresh', 'Display a spinner to indicate page is loading.', '0.25.0');
+
+INSERT INTO parameter(component, name, description_md, type, top_level, optional) SELECT 'loader-start', * FROM (VALUES
+    ('spinner', '
+The name of a [spinner](https://tabler.io/docs/components/spinners) (from tabler.io).
+Default is "spinner-border".
+Set to the empty string to disable the spinner - e.g. to display only progress
+updates.
```

Comment:
I've noticed this whole approach doesn't work inside run_sql for some reason. So I should note that 'loader-start' component should be selected before descending into run_sql.

Reply:
Yes, run_sql is a function, it returns a single json value all at once, it can't stream its results.

Reply:
I thought as much but didn't want to assume. It's good to have the clarification.